EIPS/eip-3788.md

```diff
@@ -0,0 +1,60 @@
+---
+eip: 3788
+title: Strict enforcement of chainId
+description: Reject transactions that do not explicitly have the same chainId as the node's configuration.
+author: Gregory Markou (@GregTheGreek)
+discussions-to: tbd
```

Comment:
This is a required field.  An ethereum-magicians forum thread is *strongly* recommended (you'll have to put up a bit of a fight to get other locations accepted).

Reply:
Just got to my computer, added it. Thanks for the reminder.